Bug:[chassis] ipfwd testcase add/del route command failure in multiAsics chassis

### Is it platform specific

generic

### Importance or Severity

Critical

### Description of the bug

ipfwd testcase add/del route command failure in multiAsics chassis 

This failure is caused by sonic-utils PR #4269


board201:~$ sudo ip netns exec asic0 config route add prefix 201.1.1.1/32 nexthop 10.0.0.77
Usage: config route [OPTIONS] COMMAND [ARGS]...
Try 'config route -h' for help.

Error: Missing option '-n' / '--namespace'. Choose from:
        asic0


```
if (hostname_res.is_failed or 'exception' in hostname_res) and not module_ignore_errors:
>           raise RunAnsibleModuleFail("run module {} failed".format(self.module_name), hostname_res)
E           tests.common.errors.RunAnsibleModuleFail: run module shell failed, Ansible Results =>
E           failed = True
E           changed = True
E           rc = 2
E           cmd = sudo ip netns exec asic0 config route add prefix 201.1.1.1/32 nexthop 10.0.0.77
E           start = 2026-03-30 05:45:00.875846
E           end = 2026-03-30 05:45:01.436279
E           delta = 0:00:00.560433
E           msg = non-zero return code
E           invocation = {'module_args': {'_raw_params': 'sudo ip netns exec asic0 config route add prefix 201.1.1.1/32 nexthop 10.0.0.77', '_uses_shell': True, 'expand_argument_vars': True, 'stdin_add_newline': True, 'strip_empty_ends': True, 'argv': None, 'chdir': None, 'executable': None, 'creates': None, 'removes': None, 'stdin': None}}
E           _ansible_no_log = False
E           stdout =
E           stderr =
E           Usage: config route [OPTIONS] COMMAND [ARGS]...
E           Try 'config route -h' for help.
E           
E           Error: Missing option '-n' / '--namespace'. Choose from:
E           	asic0
```

The correct command is:
board201:~$ sudo config route -n asic0 add prefix 201.1.1.1/32 nexthop 10.0.0.77
board201:~$ sudo config route -n asic0 del prefix 201.1.1.1/32 nexthop 10.0.0.77

### Steps to Reproduce

Run any testcases in ipfwd/test_dip_sip.py
or 
execute  cmd = sudo ip netns exec asic0 config route add prefix 201.1.1.1/32 nexthop 10.0.0.77

### Actual Behavior and Expected Behavior

Actual Failure:
command execution failure
Expected:
No failure

### Relevant log output

```shell

```

### Output of `show version`

```shell

```

### Attach files (if any)

_No response_

--- a/tests/ipfwd/test_dip_sip.py
+++ b/tests/ipfwd/test_dip_sip.py
@@ -40,21 +40,15 @@ def check_route(duthost, static_route, nexthop_ip, asic_id, ip_ver=4, expected_s
 
 
 def add_static_route(duthost, static_route, nexthop_ip, asic_id):
-    cli_cmd = "config route add prefix {} nexthop {}".format(static_route, nexthop_ip)
-    if duthost.is_multi_asic:
-        namespace = duthost.get_namespace_from_asic_id(asic_id)
-        cmd_prefix = "sudo ip netns exec {} ".format(namespace)
-        cli_cmd = cmd_prefix + cli_cmd
+    cli_ns_option = duthost.asic_instance(asic_id).cli_ns_option
+    cli_cmd = "config route {} add prefix {} nexthop {}".format(cli_ns_option, static_route, nexthop_ip)
     result = duthost.shell(cli_cmd)
     return result
 
 
 def delete_static_route(duthost, static_route, nexthop_ip, asic_id):
-    cli_cmd = "config route del prefix {} nexthop {}".format(static_route, nexthop_ip)
-    if duthost.is_multi_asic:
-        namespace = duthost.get_namespace_from_asic_id(asic_id)
-        cmd_prefix = "sudo ip netns exec {} ".format(namespace)
-        cli_cmd = cmd_prefix + cli_cmd
+    cli_ns_option = duthost.asic_instance(asic_id).cli_ns_option
+    cli_cmd = "config route {} del prefix {} nexthop {}".format(cli_ns_option, static_route, nexthop_ip)
     result = duthost.shell(cli_cmd)
     return result
 

